src/decisionengine/framework/dataspace/datasource.py

```diff
@@ -260,13 +260,15 @@ def close(self):
         return
 
     @abc.abstractmethod
-    def store_taskmanager(self, taskmanager_name, taskmanager_id):
+    def store_taskmanager(self, taskmanager_name, taskmanager_id, datestamp=None):
```

Comment:
Note for future edits. As a project, we use google format docstrings. If you edit one please change also the format